source/utilities.tex

```diff
@@ -5186,18 +5192,26 @@
 \throws Any exception thrown by the selected constructor of \tcode{T}.
 
 \pnum
-\remarks This constructor shall not participate in overload resolution
-unless tcode{is_constructible_v<T, Args...>} is \tcode{true}.
+\remarks
+This constructor shall not participate in overload resolution unless
+\tcode{is_copy_constructible_v<T>} is \tcode{true}
+and \tcode{is_constructible_v<T, Args...>} is \tcode{true}.
```

Comment:
You make a good point.  I was simply following existing style - I will switch to doing things your way from now on, assuming others agree.  Any one else have a preference?